packages/ui-components/src/map-legend/Bivariate.tsx

```diff
@@ -1,14 +1,177 @@
 import React, { memo } from 'react'
-import { LegendLayer } from './MapLegend'
+import cx from 'classnames'
+import { LegendLayerBivariate } from './MapLegend'
+import styles from './Bivariate.module.css'
+import BivariateArrows from './Bivariate-arrows'
 
 type BivariateLegendProps = {
-  layer: LegendLayer
+  layer: LegendLayerBivariate
   className?: string
 }
 
+/**
+ * ---------------------------------------------------------
+ * The following code is copy pasted from the MGL fork
```

Comment:
I can't think of a way of sharing this code easily, maybe duplicating and keep this comment is not that bad